src/sql/src/parsers/alter_parser.rs

```diff
@@ -74,13 +74,44 @@ impl<'a> ParserContext<'a> {
                 )));
             }
         } else if self.consume_token("MODIFY") {
-            let _ = self.parser.parse_keyword(Keyword::COLUMN);
-            let column_name = Self::canonicalize_identifier(self.parser.parse_identifier(false)?);
-            let target_type = self.parser.parse_data_type()?;
+            // Modify column type
```

Comment:
Can we use https://docs.rs/sqlparser/latest/sqlparser/parser/struct.Parser.html#method.parse_options_with_keywords

This is how sql parser parses this statement.
https://docs.rs/sqlparser/latest/src/sqlparser/parser/mod.rs.html#6961-6967

We may need to slightly change the syntax. If we only need to parse options, maybe we can refer to the implementation of this method
https://docs.rs/sqlparser/latest/src/sqlparser/parser/mod.rs.html#6412-6419

Use `parse_comma_separated()`
https://docs.rs/sqlparser/latest/sqlparser/parser/struct.Parser.html#method.parse_comma_separated

Reply:
fixed in 08a3b30da7